org-roam-db.el

```diff
@@ -206,6 +206,13 @@ This is equivalent to removing the node from the graph."
                          file))))
 
 ;;;;; Insertion
+(defun org-roam-db--insert-meta (file hash meta)
```

Comment:
Maybe make file optional, default to buffer-file-name to be in line with the signatures of the prev functions

Reply:
👍

Reply:
Actually, it's already done in `org-roam-db--update-meta`:
https://github.com/jethrokuan/org-roam/blob/46af880fbbe575d446fc8576a1b7ad374d578842/org-roam-db.el#L318